backend/sirius-web-representations/src/main/java/org/eclipse/sirius/web/representations/Success.java

```diff
@@ -12,11 +12,48 @@
  *******************************************************************************/
 package org.eclipse.sirius.web.representations;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * The status to return when the representation description handler success.
  *
  * @author gcoutable
  */
-public class Success implements IStatus {
+public final class Success implements IStatus {
```

Comment:
Do not modify that heavily this class. While switching to a builder pattern is a good solution for `CreateAccountInput` a couple days ago since it went from having 3 parameters to having 11 parameters, here you just need one additional parameter. No need to break that many classes for such change. Something like this would have way less impact:

```
public class Success implements IStatus {
  private final Map<String, Object> parameters;

  public Success() {
    this(new HashMap<>());
  }

  public Success(Map<String, Object> parameters) {
    this.parameters = Objects.requireNonNull(parameters);
  }
}
```

On top of that, such change will need to be documented in the CHANGELOD.adoc too. There are 24 files in this PR that should not be impacted by this PR and I'm not counting the other files impacted in additional products.

Reply:
Done